client/components/get-rating.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export default class GetRating extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rating: this.props.value
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const googleId = event.target.id;
+    const newRating = {
+      stars: event.target.value
+    };
+    this.setState({
+      rating: event.target.value
+    });
+    if (newRating) {
+      fetch(`/api/bookShelf/library/${googleId}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-type': 'application/json'
+        },
+        body: JSON.stringify(newRating)
+      })
+        .then(res => {
+          res.json();
+        });
+    }
```

Comment:
since `newRating` is an object, you can remove the `if`. Objects are always truthy (except `null`).